addons/elasticsearch/elasticsearch.yaml

```diff
@@ -0,0 +1,101 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: Addon
+metadata:
+  name: elasticsearch/enterprise
+  labels:
+    kubeaddons.mesosphere.io/name: elasticsearch/enterprise
+    # TODO: we're temporarily supporting dependency on an existing default storage class
+    # on the cluster, this hack will trigger re-queue on Addons until one exists.
+    kubeaddons.mesosphere.io/hack-requires-defaultstorageclass: "true"
+  annotations:
+    catalog.kubeaddons.mesosphere.io/addon-revision: "6.8.2-8"
```

Comment:
The revision number should start with 1 which would result in "6.8.2-1" if this is initially added to this repo.